traffic_ops/testing/api/v4/profiles_test.go

```diff
@@ -238,9 +239,22 @@ func UpdateTestProfiles(t *testing.T) {
 	if err != nil {
 		t.Errorf("cannot GET Profile by name: %v - %v", firstProfile.Name, err)
 	}
+	cdns, _, err := TOSession.GetCDNByName("cdn2", nil)
 	remoteProfile := resp[0]
+	oldName := remoteProfile.Name
+
 	expectedProfileDesc := "UPDATED"
+	expectedCDNId := &cdns[0].ID
+	expectedName := "testing"
+	expectedRoutingDisabled := true
+	expectedType := "TR_PROFILE"
+
 	remoteProfile.Description = expectedProfileDesc
+	remoteProfile.Type = expectedType
+	remoteProfile.CDNID = *expectedCDNId
```

Comment:
It looks like you don't need to bother making a pointer to this, as you only use the dereferenced value, not the reference itself.